src/lib/spacefinder/liveblog-right-column-adverts.ts

```diff
@@ -55,20 +53,54 @@ const fillAdvertSlots = async (numAdsToInsert: number, fromIndex: number) => {
 	}
 };
 
-// When a liveblog-right ad slot is inserted into the page, create an ad in this new slot
-export const initLiveblogRightAdverts = (): Promise<void> => {
-	if (isInVariantSynchronous(liveblogRightColumnAds, 'multiple-adverts')) {
-		return Promise.resolve();
-	}
+const createLiveblogRightAdverts = () => {
+	void fillAdvertSlots();
 
 	document.addEventListener('liveblog-right-ads-inserted', (event: Event) => {
```

Comment:
I haven't thought too hard about whether this would be simpler, but could we avoid tracking the number of adverts inserted on the Commercial side by having DCR include the id of each slot it wants filled when it fires the custom event?

Feel free to disregard this though, I don't want to make unecessary work!

Reply:
We could do, but I don't know if it would be simpler so will leave for now